code/modules/games/cards.dm

```diff
@@ -78,18 +80,59 @@
 			P.back_icon = "card_back"
 			cards += P
 
-
 	for(var/i = 0,i<2,i++)
 		P = new()
 		P.name = "joker"
 		P.card_icon = "joker"
 		cards += P
 
+/obj/item/deck/compact
+	name = "compact deck of cards"
+	desc = "A deck of playing cards. Looks like this one hasn't numbers from two to five, and jokers."
+	icon_state = "deck"
+
+/obj/item/deck/compact/LateInitialize()
```

Comment:
If it's to do a hard override without calling parent, just use a custom proc that you call from Initialize instead.